Automattic/newspack-theme
# Updates theme translations

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the theme's POT file, and generates JSON translation for two files adding the featured image options and the article subtitle textarea to single posts.

Closes #636 .

I ran into a similar issue with this PR as I did https://github.com/Automattic/newspack-blocks/pull/307, and needed to renamed the .json translation files to append `newspack` on the front (the theme's text domain), and the script's handle on the end, rather than the dynamically generated md5 hash. This whole thing is explained better in the block PR linked above.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Navigate to WP Admin > Settings > General, and switch the language to Español.
3. View the front-end of a single post, and confirm that the translations are loading -- a good spot to look is the byline, which should use 'por':

![image](https://user-images.githubusercontent.com/177561/71296940-47ead580-2336-11ea-9254-d9d14fa33af1.png)

4. Edit the post and confirm that the translation is being loaded for the JavaScript files -- the easiest way to do this is check the Featured Image Settings and confirm not all options are in English (we only have translations for 'Large' and 'Small'):

![image](https://user-images.githubusercontent.com/177561/71296972-69e45800-2336-11ea-818f-c57f31a08267.png)

5. Navigate to WP Admin > Settings > General, and switch the site language to Português do Brasil.
6. View the front end of a single post, and confirm the translations are loading -- I good spot again to look is the byline:

![image](https://user-images.githubusercontent.com/177561/71297075-d4959380-2336-11ea-8148-491524acf2a4.png)

7. Edit the post and confirm that the translation is being loaded for the JavaScript files -- the Featured Image Settings should also not all be in English (we only have translations for 'Large' and 'Small'):

![image](https://user-images.githubusercontent.com/177561/71297107-f5f67f80-2336-11ea-9036-7ca4c31deb3f.png)

### Other information:
